apps/callscreen/js/handled_call.js

```diff
@@ -455,10 +440,26 @@ HandledCall.prototype.connected = function hc_connected() {
 
 HandledCall.prototype.disconnected = function hc_disconnected() {
   if (this._leftGroup) {
-    CallScreen.showStatusMessage({
-      id: 'caller-left-call',
-      args: { caller: this._cachedInfo.toString() }
-    });
+    if (typeof(this._cachedInfo) === 'string') {
+      CallScreen.showStatusMessage(this._cachedInfo);
+    } else if (this._cachedInfo.id) {
+      var id;
+
+      switch (this._cachedInfo && this._cachedInfo.id) {
+        case 'withheld-number': id = 'withheld-number-left-call';  break;
+        case 'voiceMail':       id = 'voicemail-left-call';        break;
+      }
+
+      CallScreen.showStatusMessage({
+        id: id,
+        args: { caller: this._cachedInfo ? this._cachedInfo.toString() : '' }
```

Comment:
That's a leftover from the old code, there's no need to pass an id/args couple there, I'll remove it.